packages/react-components/src/templates/EventsPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { perRem } from '../pixels';
+import EventsPageHeader from './EventsPageHeader';
+import { contentSidePaddingWithNavigation } from '../layout';
+
+const articleStyles = css({
+  alignSelf: 'stretch',
```

Comment:
I feel like something will need a change of defaults :D almost everywhere has this alignment